cheroot/test/test_conn.py

```diff
@@ -1017,12 +963,11 @@ def test_Content_Length_out(
     ))
 
 
-@pytest.mark.xfail(
```

Comment:
@cyraxjoe this was failing but flakily. I was unable to reproduce this on every run.